sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -206,7 +206,7 @@ abstract class QueryTest extends PlanTest {
     val jsonString = try {
       logicalPlan.toJSON
     } catch {
-      case e =>
+      case e: Throwable =>
```

Comment:
This is more a potential bug in the original code, but why not use `NonFatal(e)` here instead?